nucypher/blockchain/eth/actors.py

```diff
@@ -340,7 +338,8 @@ def save_deployment_receipts(self, transactions: dict) -> str:
             for contract_name, transactions in transactions.items():
                 contract_records = dict()
                 for tx_name, txhash in transactions.items():
-                    receipt = {item: str(result) for item, result in self.blockchain.wait_for_receipt(txhash).items()}
+                    receipt = self.blockchain.client.w3.eth.waitForTransactionReceipt(txhash)
```

Comment:
`Web3Client` has a `sendTransaction` method - should we add `waitForTransactionReceipt` to the client as well? My preference would be to obfuscate `w3.eth`

Reply:
:heavy_check_mark: